src/api/moderation.js

```diff
@@ -27,3 +27,46 @@ export const setSubmitStatus = (id, Status) => {
     return {};
   }
 };
+
+
+export const setClusters = () => {
+  try {
+    return apiAuthPut('/mod/clusters').then(
+      res => {
+        console.log(res);
+        return res;
+      }
+    );
+  } catch (err) {
+    console.log(err);
+    return {err};
+  };
+};
+
+export const setFaceoffs = () => {
```

Comment:
Clean code. Would like some comments to understand more fully what the purpose of this and other API endpoints are.